File: libraries/hdm/functions/types/response/SnmpGetParameterValuesResponseDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.response.SnmpGetParameterValuesResponseDTO') //< public
//> needs hdm.functions.types.struct.SnmpGetParameterStructDTO
.satisfies('hdm.functions.types.response.ResponseDTO')
.satisfies('java.io.Serializable')
.protos({
    parameterList:null, //< private SnmpGetParameterStructDTO[]
    //> public SnmpGetParameterStructDTO[] getParameterList()
    getParameterList:function(){
        return this.parameterList;
    },
    //> public void setParameterList(SnmpGetParameterStructDTO[] parameterList)
    setParameterList:function(parameterList){
        this.parameterList=parameterList;
    }
})
.endType();

File: libraries/hdm/functions/types/response/SnmpGetParameterValuesResponseDTO.java
package hdm.functions.types.response;

import hdm.functions.types.struct.SnmpGetParameterStructDTO;
import java.io.Serializable;

public class SnmpGetParameterValuesResponseDTO
  implements ResponseDTO, Serializable
{
  private SnmpGetParameterStructDTO[] parameterList;

  public SnmpGetParameterStructDTO[] getParameterList()
  {
    return this.parameterList;
  }

  public void setParameterList(SnmpGetParameterStructDTO[] parameterList)
  {
    this.parameterList = parameterList;
  }
}